interface/forms/eye_mag/SpectacleRx.php

```diff
@@ -444,239 +410,241 @@ function delete_me(delete_id) {
-                                            </table>
-                                            <?php
-                                } else {
-                                    if (!empty($row['ODADD']) || !empty($row['OSADD'])) {
-                                        $adds = 1;
-                                    } else {
-                                        $adds = '';
-                                    }
+                        <div class="table-responsive">
+                            <table style="margin:2px auto;">
+                                <tr>
+                                    <td class="right bold" style="width:250px;"><b><?php echo xlt('RX Print Date'); ?>: </b></td>
+                                    <td>&nbsp;&nbsp;<?php echo text($row['date']); ?></td>
+                                </tr>
+                                <tr>
+                                    <td class="right bold"><b><?php echo xlt('Visit Date'); ?>: </b></td>
+                                    <td>&nbsp;&nbsp;<?php echo text($row['REFDATE']); ?></td>
+                                </tr>
+                                <tr>
+                                    <td class="right bold"><b><?php echo xlt('Expiration Date'); ?>: </b></td>
+                                    <td>&nbsp;&nbsp;<?php
+                                        echo text($expir_date);
                                     ?>
-                                            <table id="CTLRx" name="CTLRx" class="refraction">
-                                                <tr>
-                                                    <td colspan="4"
-                                                        class="bold left text-uppercase text-top" style="display: flex;
-                                            align-items:top"><u><?php echo xlt('Right Lens'); ?></u>
-                                                    </td>
-                                                </tr>
-                                                <tr class="bold underline">
-                                                    <td></td>
-                                                    <td><?php echo xlt('Sph{{Sphere}}'); ?></td>
-                                                    <td><?php echo xlt('Cyl{{Cylinder}}'); ?></td>
-                                                    <td><?php echo xlt('Axis{{Axis in a glasses prescription}}'); ?></td>
-                                                    <td><?php echo xlt('BC{{Base Curve}}'); ?></td>
-                                                    <td><?php echo xlt('Diam{{Diameter}}'); ?></td>
-                                            <?php
-                                            if ($adds) {
+                                    </td>
+                                </tr>
+
+                                <tr>
+                                    <td class="right bold"><?php echo xlt('Refraction Method'); ?>:</td>
```

Comment:
In this entire file, replace the deprecated `bold` class with `font-weight-bold`.